doc/source/changes/version_0_28.rst.inc

```diff
@@ -255,8 +255,20 @@ Miscellaneous improvements
 
   Closes :issue:`488`.
 
-* deprecated `nan_equal` function in favor of `equal` function which has the same optional arguments
-  as the `LArray.equals` method (closes :issue:`593`).
+* deprecated `nan_equal` function in favor of `equals` function. The `equals` function has the same optional
+  arguments as the `LArray.equals` method but compares two arrays element-wise and returns array of booleans:
+
+      >>> equals(arr1, arr2)
+      a     a0     a1
+         False  False
+      >>> equals(arr1, arr2, atol=0.01)
+      a    a0    a1
+         True  True
+      >>> equals(arr1, arr2, rtol=0.01)
```

Comment:
given you present the function as a replacement for nan_equal, you should have at least one example with nans and explain or show how to replicate nan_equal behavior

Reply:
OK but seriously, has `nan_equal` ever been used at least once by one of our current users? 
Seems like this function was only there to be used in larray-editor project (when comparing arrays/sessions).

Reply:
Not that I know of, indeed. But then you can not show any example at all. I was just pointing out that your way to presented things was inconsistent.